package.json

```diff
@@ -51,7 +51,7 @@
     "postcss-reporter": "^1.3.0",
     "postcss-resolve-nested-selector": "^0.1.1",
     "postcss-scss": "^0.1.3",
-    "postcss-selector-parser": "^2.0.0",
+    "postcss-selector-parser": "2.1.0",
```

Comment:
@davidtheclark Did explicitly want to _pin_ `postcss-selector-parser` to version `2.1.0` here?

This will probably cause a merge conflict with the v7 change here https://github.com/stylelint/stylelint/commit/90b9e3e5d84b4f46224b59c6362b62f33a7cf2cf